### PR TITLE
docs(showcase): Example now is more easy to read and understand

### PR DESCRIPTION
**Pull request by GabrielRodriguesS**
Example to calendar with notices now is a format closer to its actual use in production.
This change is for version 1.2.0 of the API, the examples for the previous versions I can arrange in the coming weeks if this pull-request is accepted.

The example it's running here: https://stackblitz.com/edit/angular-nz-calendar

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[*] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: components/calendar/demo/notice-calendar.ts [hunk 1]
```diff
@@ -3,84 +3,81 @@ import { Component } from '@angular/core';
...
-        </ng-container>
-      </ul>
-      <ng-container *nzMonthCell="let month">
-        <div *ngIf="getMonthData(month) as monthData" class="notes-month">
-          <section>{{ monthData }}</section>
-          <span>Backlog number</span>
-        </div>
+<nz-calendar>
+   <ul *nzDateCell="let date" class="events">
+      <ng-container *ngFor="let event of getEvents(date)">
```
[the remark this answers is not in the document]
Comment (GabrielRodriguesS) [reply components/calendar/demo/notice-calendar.ts]: I searched, but don't found. When I use  "let date" I'm get the date from where?

[the remark this answers is not in the document]
Comment (vthinkxie) [reply components/calendar/demo/notice-calendar.ts]: I mean you should add trackBy function to  `let event of getEvents(date)`

File: components/calendar/demo/notice-calendar.ts [hunk 2]
```diff
@@ -3,84 +3,79 @@ import { Component } from '@angular/core';
...
+    type: 'error', content: 'This is error event', title: 'This is title to error event',
+    date: this.yesterday
+  },
+  {
+    type: 'default', content: 'This is default event', title: 'This is title to default event',
+    date: this.nextMonth
+  }];
+ 
+  getEvents(index, event) {
+    event.date.setHours(0, 0, 0, 0);
```
Comment (vthinkxie) [inline components/calendar/demo/notice-calendar.ts]: should not set data in trackByFn

Comment (GabrielRodriguesS) [reply components/calendar/demo/notice-calendar.ts]: When I did this example, I did not notice it, thank you, it's corrected.

File: components/calendar/demo/notice-calendar.ts [hunk 3]
```diff
@@ -3,84 +3,83 @@ import { Component } from '@angular/core';
...
+  current = new Date();
+  tomorrow = new Date();
+  yesterday = new Date();
+  nextMonth = new Date();
+
+  constructor() {
+    this.tomorrow.setDate(this.tomorrow.getDate() + 1);
+    this.yesterday.setDate(this.yesterday.getDate() - 1);
+    this.nextMonth.setMonth(this.nextMonth.getMonth() + 1);
+    // Ignore time
```
Comment (GabrielRodriguesS) [inline components/calendar/demo/notice-calendar.ts]: Code added to ignore the time in the attributes.

**Comment by vthinkxie** [comment]
Due to the version iteration, this issue will be closed. Please contact us if you have any questions, thank you for your pr!

**Pull request closed without merging**